shared/profile/dumb.js

```diff
@@ -316,16 +316,24 @@ const dumbConfirmOrPendingMap: DumbComponentMap<ConfirmOrPending> = {
     'Confirm Twitter': confirmBase,
     'Confirm Reddit': {...confirmBase, platform: 'reddit'},
     'Confirm GitHub': {...confirmBase, platform: 'github'},
-    'Pending Hacker News': {...confirmBase, ...pending,
+    'Pending Hacker News': {...confirmBase,
+      ...pending,
       platform: 'hackernews',
       message: 'Hacker News caches its bios, so it might be a few hours before you can verify your proof. Check back later.'},
     'Confirm Coinbase': {...confirmBase, platform: 'coinbase'},
-    'Confirm Bitcoin': {...confirmBase, platform: 'btc', usernameSubtitle: undefined,
-      message: 'You Bitcoin address has now been signed onto your profile.', title: 'Verified'},
-    'Pending dns': {...confirmBase, ...pending,
-      platform: 'dns', usernameSubtitle: 'dns',
+    'Confirm Bitcoin': {...confirmBase,
+      platform: 'btc',
+      usernameSubtitle: undefined,
+      message: 'You Bitcoin address has now been signed onto your profile.',
```

Comment:
Guess it does make the diffs easier to read :)

On Friday, August 19, 2016, Chris Ball <notifications@github.com> wrote:

> In shared/profile/dumb.js
> <https://github.com/keybase/client/pull/3972#discussion_r75564429>:
>
> >        platform: 'hackernews',
> >        message: 'Hacker News caches its bios, so it might be a few hours before you can verify your proof. Check back later.'},
> >      'Confirm Coinbase': {...confirmBase, platform: 'coinbase'},
> > -    'Confirm Bitcoin': {...confirmBase, platform: 'btc', usernameSubtitle: undefined,
> > -      message: 'You Bitcoin address has now been signed onto your profile.', title: 'Verified'},
> > -    'Pending dns': {...confirmBase, ...pending,
> > -      platform: 'dns', usernameSubtitle: 'dns',
> > +    'Confirm Bitcoin': {...confirmBase,
> > +      platform: 'btc',
> > +      usernameSubtitle: undefined,
> > +      message: 'You Bitcoin address has now been signed onto your profile.',
>
> Typo: Your
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3972/files/6bc8ea0d2ad64e9bc5b601e8d21d72c0170f0096#r75564429>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAfhUwmqEbNtI5UVubTJehfz1hZ4WJvdks5qhjpbgaJpZM4Jo4ag>
> .
>